benchmark/atoi.cpp

```diff
@@ -52,16 +58,96 @@ auto lemire_as_zoo_swar(const char *chars) {
     return uint32_t(by10001base2to32.value() >> 32);
 }
 
+std::size_t spaces_glibc(const char *ptr) {
+    auto rv = 0;
+    while(isspace(ptr[rv])) { ++rv; }
+    return rv;
+}
+
 namespace zoo {
 
+//constexpr
```

Comment:
Raises the question of why it isn't constexpr

Reply:
It would not be exportable.
Also, all of SIMD support is un-usable in constexpr.  I should delete the comment, but that's the reason why we have to dial down constexpr quite a lot in many parts of the library.